param/_utils.py

```diff
@@ -46,6 +50,10 @@ class ParamFutureWarning(ParamWarning, FutureWarning):
     Always displayed.
     """
 
+class Skip(RuntimeError):
```

Comment:
```suggestion
class SkipReferenceUpdate(RuntimeError):
```

Maybe be more explicit about what `Skip` does; this is only a suggestion.

Reply:
This will be user level as, so I like the short name here.

Reply:
I think it is too ambiguous to be useful. 

If it is a public API, I think it should be mentioned somewhere in the documentation.